### ISSUE TITLE
Alt text not included in images from API

### ISSUE THREAD

**Issue by danielpost** [Shopify/js-buy-sdk#422]
Hi,

I'm not sure if I'm just missing it, but it seems like the alt text is not available when querying products. Is there any way to retrieve this?

Cheers!

**Comment by minasmart** [comment]
not sure what you mean by alt text. do you mean on images?

**Comment by danielpost** [comment]
Yes, this one I mean:

![image](https://user-images.githubusercontent.com/1713699/30867237-b92e6992-a2db-11e7-8423-47ef19c38238.png)
